### PR TITLE
Allow single character abbreviations

### PR DESCRIPTION
**Pull request by MihaZupan**

### CODE REVIEW

File: src/Markdig.Tests/Specs/AbbreviationSpecs.md [hunk 1]
```diff
@@ -97,4 +97,13 @@ Abbreviations should only match when surrounded by whitespace:
 PRAA
 .
 <p>PRAA</p>
+````````````````````````````````
+
+One-character abbreviations should be allowed
```
Comment (xoofx) [inline src/Markdig.Tests/Specs/AbbreviationSpecs.md]: You need to save the `Specs.tt` file from VS in order to regenerate the tests (same for #292)

Comment (MihaZupan) [reply src/Markdig.Tests/Specs/AbbreviationSpecs.md]: I know, but didn't want to commit 20k lines of change again. A lot of new lines appear whenever I regenerate the spec.

Comment (xoofx) [reply src/Markdig.Tests/Specs/AbbreviationSpecs.md]: It is fine, otherwise the tests are not running. I don't check the changes in this file

**Comment by MihaZupan** [comment]
Closing as related to #294

**Pull request closed without merging**